VAMobile/src/constants/common.ts

```diff
@@ -57,6 +57,9 @@ export const MAIL_TO_REGEX_EXP = new RegExp(
 export const PHONE_REGEX_EXP = new RegExp(
   /^\s*(?:\+?(\d{0,3}))?[-. (]*(\d{3})[-. )]*(\d{3})[-. ]*(\d{4})(?: *(x)(\d+))?,?.?\s*$/,
 )
+
+export const SSN_REGEX_EXP = new RegExp(/^\d{3}-?\d{2}-?\d{4}$/)
```

Comment:
export const SSN_REGEX_EXP = new RegExp(/^\d{3}-?\s?\d{2}-?\s?\d{4}$/)

Reply:
this is the regex for those changes to include spaces as part of the regex search

Reply:
I'm actually going to just make these both strict '-' searches currently and we can spin off more in-depth PII filtering at a later date if it's necessary.